### ISSUE TITLE
Idea: Admin container should be customizable in theme

### ISSUE THREAD

**Issue by chlupac** [oqtane/oqtane.framework#561]
currently is assigned in ContainerBuilder.razor
```
        _moduleState = Module; // passed in from Pane component
        string container = _moduleState.ContainerType;
        if (PageState.ModuleId != -1 && PageState.Action != "" && _moduleState.UseAdminContainer)
        {
            container = Constants.DefaultAdminContainer;
        }
```
can be easily replaced with custom implementation from theme.

**Comment by sbwalker** [comment]
note that if you set UseAdminContainer to false it will use the container assigned to the module rather than the defaultadmincontainer

**Comment by sbwalker** [comment]
The problem with the UseAdminContainer property is that it must be set through code - which is fine if you are developing your own custom modules, but not useful when you are using pre-existing modules. I am guessing that you are looking for the ability for the site Administrator to be able to set an option through the UI to control this behavior ie. a Default Admin Container at the Site level?

**Comment by chlupac** [comment]
I solved my problem with developing custom dialog component and set  DefaultAdminContainer=false
Would be nice to set admin container on site level, but this can have impact to admin components. Idea #561 is probably wrong idea.

**Comment by sbwalker** [comment]
implemented as part of #1088
